pkg/ctl/enable/profile.go

```diff
@@ -44,11 +44,11 @@ func (opts options) validate() error {
 	return nil
 }
 
-func applyGitops(cmd *cmdutils.Cmd) {
+func enableProfile(cmd *cmdutils.Cmd) {
```

Comment:
Let's stick to the convention...
```suggestion
func enableProfileCmd(cmd *cmdutils.Cmd) {
```